src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java

```diff
@@ -287,24 +279,46 @@ private static String formatMessageBatchToStore(GroupedMessageBatchToStore batch
      */
     private void verifyBatch(T delivery) {
         List<M> messages = getMessages(delivery);
+        HashMap<SessionKey, SessionData> innerCache = new HashMap<>();
```

Comment:
when is the innerCache applied to global cache?

Reply:
in the handle method after calling verifyBatch all messages are added to global cache